aries-site/src/layouts/content/ContentSection.js

```diff
@@ -16,7 +16,10 @@ export const ContentSection = ({ children, lastSection }) => {
           ? { bottom: size !== 'small' ? 'large' : 'xlarge' }
           : undefined
       }
-      pad={{ bottom: size !== 'small' ? 'large' : 'xlarge' }}
+      pad={{
+        bottom: size !== 'small' ? 'large' : 'xlarge',
+        horizontal: 'xxsmall',
```

Comment:
```suggestion
        horizontal: 'xxsmall',
        top: 'xxsmall',
```

Reply:
Wondering if we should add this here now. I noticed when I was changing the topic to an anchor that it was getting cut off on the top and the left side.

Reply:
oooh. Great point. Done.